assets/js/app/llms-forms.js

```diff
@@ -378,12 +377,49 @@ LLMS.Forms = {
 			this.$states.html( '<option>&nbsp</option>' );
 			this.disable_field( this.$states );
 		} else {
+			var placeholder = this.get_state_placeholder( this.$states, country_code );
 			this.enable_field( this.$states );
 			this.$states.html( opts );
+			if ( '' !== placeholder ) {
+				this.set_state_placeholder( this.$states, placeholder );
+			}
 		}
 
 	},
 
+    /**
+	 * Get the state placeholder and/or replace with {REGION}
+	 *
+	 * @since [version]
+	 * 
+	 * @param {String} country_code Currently selected country code. 
+	 * @return {String} placeholder text if any has been set, otherwise returns default state/region. 
+	 */
+	get_state_placeholder: function( holder, country_code ) {
+
+		var placeholder;
+		
+		if ('undefined' === typeof this.address_info[ country_code ] ) {
+			placeholder = holder.attr( 'placeholder' ).replace( '{REGION}', LLMS.l10n.translate( 'State/Region' ));
+		} else {
+			placeholder = holder.attr( 'placeholder' ).replace( '{REGION}', this.address_info[country_code].state );
+		}
+
+		return placeholder;
+
+		},
+	
+	/**
+	 * Set the state placeholder to the state field
+	 *
+	 * @since [version]
+	 * @param {String} placeholder Sets the 'placeholder' text to the field so users must select their state/region. 
```

Comment:
```suggestion
	 * @since [version]
	 *
	 * @param {String} placeholder Sets the 'placeholder' text to the field so users must select their state/region. 
```

Empty line should come between `@since` and `@param` blocks.